build/common.ps1

```diff
@@ -438,12 +438,14 @@ Function Test-XProject {
                     $opts += '-v'
                 }
                 $opts += 'test', '--configuration', $Configuration, '--framework', 'netcoreapp1.0'
+                if ($VerbosePreference) {
+                    $opts += '-verbose'
+                }
 
                 Trace-Log "$DotNetExe $opts"
-
                 & $DotNetExe $opts
 
-                if (-not $?) {
+                if ($LASTEXITCODE -ne 0) {
```

Comment:
`$?` is a magic PS variable evaluating outcome of a process run including last exit code, error output, warnings, etc. This introduces a lot of noise on AppVeyor VM. Generally checking last exit code is enough for almost every build tool we use.